server/sonar-web/src/main/js/apps/code/components/App.js

```diff
@@ -179,6 +181,8 @@ class App extends React.PureComponent {
 
     return (
       <div className="page page-limited">
+        <Helmet title={translate('code')} />
```

Comment:
better use `code.page`, because to make sure the tab title exactly matches the page title whatever the location is.